### PR TITLE
[DE] Update `Internet_Relay_Chat`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Internet_Relay_Chat/de.md [hunk 1]
```diff
@@ -1,69 +1,85 @@
----
-outdated: true
----
-
 # Internet Relay Chat
 
-[IRC](https://de.wikipedia.org/wiki/Internet_Relay_Chat) ist ein Chat-System, dem es einen möglich macht, mit anderen Usern sich in einem Channel zu unterhalten. Hierfür gibt es verschiedene Applikationen und Add-Ons.
+*Für mehr Informationen, siehe: [Internet Relay Chat (Wikipedia)](https://en.wikipedia.org/wiki/Internet_Relay_Chat)*
```
Comment (respektive) [inline wiki/Internet_Relay_Chat/de.md:3]:
```suggestion
*Für mehr Informationen, siehe: [Internet Relay Chat (Wikipedia)](https://de.wikipedia.org/wiki/Internet_Relay_Chat)*
```
would probably link to the german site here and remove the link in next line

File: wiki/Internet_Relay_Chat/de.md [hunk 2]
```diff
@@ -1,69 +1,85 @@
...
 
-## osu! Chat
+[IRC](https://de.wikipedia.org/wiki/Internet_Relay_Chat) ist ein Protokoll der Anwendungsschicht, das das Chatten mit anderen Benutzern ermöglicht.
```
Comment (respektive) [inline wiki/Internet_Relay_Chat/de.md:5]:
```suggestion
IRC ist ein Protokoll auf Anwendungsebene, das das Chatten mit anderen Benutzern ermöglicht.
```

File: wiki/Internet_Relay_Chat/de.md [hunk 3]
```diff
@@ -1,69 +1,85 @@
...
 
-In osu! ist es möglich mit dem eigenen IRC Client sich zu dem Chat-Server zu verbinden, um über diesen Weg in den Channels mit anderen Usern zu schreiben. Selbst dann, wenn bei euch osu! nicht im Hintergrund läuft. Allerdings fehlen ein paar Funktionen, da Bancho eigene Einstellungen verwendet, daher könnten wohlmöglich ein paar Funktionen des eigenen IRC Clients entfallen.
+## osu!Bancho
 
-### Zum Server verbinden
+*Hinweis: dieser Server stellt keine vollständige Implementierung des IRC-Protokolls bereit. Einige komplexere Funktionen deines IRC-Clients (z. B. [HexChat](https://hexchat.github.io/)) funktionieren nur bedingt oder gar nicht.*
 
-Falls ihr einen Client habt und über diesen euch verbinden wollt, verbindet euch zu [cho.ppy.sh](irc://cho.ppy.sh) oder [irc.ppy.sh](irc://irc.ppy.sh) (beide sind verwendbar) zum Port **6667** (dem Standard IRC Port).
+*osu!Bancho* (manchmal abgekürzt einfach nur *Bancho*) bietet ein Gateway für dne Zugriff auf IRC (Chat im Spiel). Du kannst jeden modernen IRC-Client nutzen, um dich damit zu verbinden (scrolle herunter für eine Liste der populärsten Clients).
```
Comment (respektive) [inline wiki/Internet_Relay_Chat/de.md:11]:
```suggestion
*osu!Bancho* (manchmal abgekürzt einfach nur *Bancho*) bietet ein Gateway für den Zugriff auf IRC (Chat im Spiel). Du kannst jeden modernen IRC-Client nutzen, um dich damit zu verbinden (scrolle herunter für eine Liste der populärsten Clients).
```

File: wiki/Internet_Relay_Chat/de.md [hunk 4]
```diff
@@ -1,69 +1,85 @@
...
 
-```
-* Welcome to osu!bancho.
-* -
-* - You are required to authenticate before accessing this service.
-* - Please click the following link to complete this process:
-```
+- Server: `irc.ppy.sh`
+- Port: `6667`
+- Benutzername: dein osu! Benutzername. Ersetze Leerzeichen durch Unterstriche (z. B. `beppy master 1000` wird zu `beppy_master_1000`)
```
Comment (respektive) [inline wiki/Internet_Relay_Chat/de.md:19]:
```suggestion
- Benutzername: dein osu! Benutzername. Ersetze Leerzeichen durch Unterstriche (z. B. wird `beppy master 1000` zu `beppy_master_1000`)
```

File: wiki/Internet_Relay_Chat/de.md [hunk 5]
```diff
@@ -1,69 +1,85 @@
...
+| `/ignore <Benutzername>` | Ignoriere einen Benutzer (Nachrichten werden versteckt) |
+| `/away <Nachricht>` | Lasse eine Abwesenheitsnachricht für jeden da, der dich versucht zu kontaktieren |
+| `/away` | Lösche die Abwesenheitsnachricht |
+| `/query <Benutzername>` | Öffne einen Chat mit Benutzername (ersetze Leerzeichen durch Unterstriche) |
 
-`Um euch dauerhaft zu authentisieren, ändert euer IRC Passwort (Server Passwort) zu: XXXXXXX`
+### Join/Part/Quit Nachrichten deaktivieren
 
-## Grundlegende IRC Kommandos
+Normalerweise informieren dich viele IRC-Clients, sobald ein Benutzer einen Channel verlässt. Für Netzwerke wie osu!Bancho mit tausenden von Nutzern, die sich ständig ein- und ausloggen, können solche Fluten an Nachrichten nervig werden. Im Folgenden findest du eine Liste an populären IRC-Clients und wie du diese Nachrichten dort deaktivieren kannst:
```
Comment (respektive) [inline wiki/Internet_Relay_Chat/de.md:38]:
```suggestion
Normalerweise informieren dich viele IRC-Clients, sobald ein Benutzer einen Channel verlässt. Für Netzwerke wie osu!Bancho mit tausenden von Nutzern, die sich ständig ein- und ausloggen, können solche Fluten an Nachrichten nervig werden. Im Folgenden findest du eine Liste gängiger IRC-Clients und wie du diese Nachrichten dort deaktivieren kannst:
```

File: wiki/Internet_Relay_Chat/de.md [hunk 6]
```diff
@@ -1,69 +1,85 @@
...
+## FAQ
 
-Jedes Mal wenn jemand dem Channel betretet oder verlässt, kommt folgende Nachricht:
+### Ich kann mich nicht einloggen
 
-```
-someuser has joined #somechannel
-someuser has quit #somechannel
-```
+osu!Bancho benutzt Klartext bei der Authentifizierung — stelle sicher, dass kein spezieller Authentifizierungsmodus ausgewählt ist in den Einstellungen deines IRC-Clients.
```
Comment (respektive) [inline wiki/Internet_Relay_Chat/de.md:55]:
```suggestion
osu!Bancho benutzt Klartext bei der Authentifizierung — stelle sicher, dass in den Einstellungen deines IRC-Clients kein spezieller Authentifizierungsmodus ausgewählt ist.
```

File: wiki/Internet_Relay_Chat/de.md [hunk 7]
```diff
@@ -1,69 +1,85 @@
...
-| [mIRC](https://www.mirc.com/) | Tools » Options » wählt "IRC". Klickt auf den "Events..." Button. Ändert die "joins", "parts", "quits", und "nicks" zu der jeweiligen Einstellung: "In Status" oder "Hide" sind gute Optionen. [1](http://web.archive.org/web/20160304201229/http://i.clintecker.com/disable-irc-msgs.html). |
-| [Quassel IRC](https://quassel-irc.org/) | Rechtsklick auf das Fenster worin ihr chattet, wählt dann "Hide Events" » "Join/Part/Quit". |
-| [XChat](http://xchat.org/) | Rechtsklick auf den Tab, den ihr ändern wollt. Im Untermenu der Channel Namen, könnt ihr "Show join/part messages" ein- und ausschalten. Oder ihr gibt `/set irc_conf_mode 1` ein [2](http://xchat.org/faq/#q211), um diese Nachrichten über alle Channels hindurch zu deaktivieren. |
-
-Sofern euer IRC Client nicht verzeichnet ist, könnt ihr diese hier hinzufügen und wie es bei diesen funktioniert.
+Versuche das Folgende:
+
+1. Vergewissere dich, dass du das richtige Passwort von der [IRC-Authentifizierungsseite](https://osu.ppy.sh/p/irc) benutzt.
+2. Wenn dein Benutzername Leerzeichen enthält, ersetze diese bitte mit Unterstrichen (z. B. `Dieser Benutzername` wird zu `Dieser_Benutzername`).
```
Comment (respektive) [inline wiki/Internet_Relay_Chat/de.md:64]:
```suggestion
2. Wenn dein Benutzername Leerzeichen enthält, ersetze diese bitte mit Unterstrichen (z. B. wird `Dieser Benutzername` zu `Dieser_Benutzername`).
```

File: wiki/Internet_Relay_Chat/de.md [hunk 8]
```diff
@@ -1,69 +1,85 @@
...
+Tippe die ersten paar Buchstaben des Benutzernamens ein und drücke dann `Tab`, um die Autovervollständigung durchzugehen.
+
+### Warum sind manche Namen mit unterschiedlichen Präfixen versehen?
+
+Der IRC-Standard hat ein Channelmoduskonzept oder Aktionen, die jeder Nutzer ausführen kann. In osu!Bancho werden zwei Modi für spezielle Gruppen von Benutzern verwendet:
+
++ `+` oder "voice status": der Benutzer ist via einem externen IRC-Client verbunden
+- `@` oder "chat operator status": der Benutzer ist ein Chatmoderator ([GMT](/wiki/People/The_Team/Global_Moderation_Team) oder [NAT](/wiki/People/The_Team/Nomination_Assessment_Team))
+
+Benutzer, die über den osu!-Client verbunden sind, haben kein Präfix.
```
Comment (respektive) [inline wiki/Internet_Relay_Chat/de.md:81]:
```suggestion
Benutzer, die über den osu!-Client oder die Website verbunden sind, haben kein Präfix.
```

**Review by respektive — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**